kogito-usertasks-with-security-oidc-keycloak/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0"?>
+<project
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.kogito.quickstarts</groupId>
+    <artifactId>kogito-quickstarts</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kogito-usertasks-with-security-oidc-keycloak</artifactId>
+
+  <name>kogito-usertasks-with-security-oidc-keycloak</name>
+  <description>Kogito user tasks orchestration with security enabled on REST api - open id connect adapter(keycloak)</description>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-bom</artifactId>
+        <version>${quarkus.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-oidc</artifactId>
+    </dependency>
```

Comment:
so in the end I was wrong ... the quarkus-jsonp is needed... not for kogito but for JWT handling.. otherwise it fails with

````
Caused by: java.lang.ClassNotFoundException: javax.json.JsonObject
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
````

so please add it back. Sorry for the trouble.

Reply:
@mswiderski I swear today at my environment it works without this dep! But adding it because yesterday I believed it was 'ultra'- necessary! :S

Reply:
hehe sounds like a good enough reason to me :)